packages/api/src/models/team.ts

```diff
@@ -9,6 +9,7 @@ export interface ITeam {
   allowedAuthMethods?: 'password'[];
   apiKey: string;
   hookId: string;
+  collectorAuthenticationEnforced: boolean;
```

Comment:
this enables collector auth moving forward, but preserves existing v2 no-auth behavior for existing deployments